js/adapt-stateful-session.js

```diff
@@ -195,33 +218,12 @@ define([
 		},
 
 		submitAssessmentFailed: function() {
-			if (this._config && this._config._reporting.hasOwnProperty("_onAssessmentFailure")) {
-				var onAssessmentFailure = this._config._reporting._onAssessmentFailure;
-				if (onAssessmentFailure === "") return;
-					
-				Adapt.offlineStorage.set("status", onAssessmentFailure);
+			if (this._config && this._config._reporting._onAssessmentFailure) {
```

Comment:
I don't _think_ this function is needed any more as it's now covered by `onTrackingComplete`... just doing a bit of testing to make sure that's the case though

Reply:
The `tracking:complete` event does indeed cause the correct status to be sent.  

I didn't take this function out though as it's bound to assessment completion.  If the assessment was at the start or the middle of the course, for example, it might be beneficial to have this written back to the LMS while the user continues progress on the course?  Also, since the `score` is written in the same chain of events I thought it best to leave this as is.

Reply:
This definitely isn't needed anymore.